spaces/Ourbloc/index.json

```
@@ -0,0 +1,15 @@
+{
+  "key": "Ourbloc",
```

The key will be the url and we want it all in lowercase. Please modify the value of the key by something like `ourbloc` and your space directory too. Thank you.
```suggestion
  "key": "ourbloc",
```